pkg/kyverno/test/render.go

```diff
@@ -0,0 +1,38 @@
+package test
```

Comment:
why is this file called `render.go`?

Reply:
changed to git.go, as its specific to clone and list yamls from git